src/tests/multihost/alltests/test_sudo.py

```diff
@@ -110,3 +110,74 @@ def test_timed_sudoers_entry(self,
                 multihost.master[0].run_command(journalctl_cmd)
                 pytest.fail("%s cmd failed for user %s" % ('sudo -l', 'foo1'))
             ssh.close()
+
+    @pytest.mark.tier2
+    def test_randomize_sudo_timeout(self, multihost,
+                                    backupsssdconf, sudo_rule):
+        """
+        :title: sudo: randomize sudo refresh timeouts
+        :id: 57720975-29ba-4ed7-868a-f9b784bbfed2
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1925514
+        :customerscenario: True
+        :steps:
+          1. edit sssdconfig and specify sssd smart, full timeout option
```

Comment:
First words should be in capital edit -> Edit

Reply:
fixed it.